tools/releaseBuild/macOS/vsts.yml

```diff
@@ -0,0 +1,26 @@
+steps:
+- task: ShellScript@2
```

Comment:
Curious, why all tasks here have the same name?

Reply:
This is the name of the `<type>@<version>` of the task .   We are running the same task with different parameters.  So the name is the same.

Reply:
A comment would be great.